### PR TITLE
Comment to RunClass

### PR DESCRIPTION
**Pull request by JoonaPeltola**
Added comment to RunClass

### CODE REVIEW

**Review by konstahakli — CHANGES_REQUESTED** [review]
Lookin great but more comments!

File: RunClass.java [hunk 1]
```diff
@@ -17,7 +17,7 @@ public static void main(String[] args) {
 		LFObj = new LineFollower(DE);
 		LSObj = new lightSensor(DE);
 
-		
+		//starting threads
```
Comment (Tuomas-Talkio) [inline RunClass.java:20]:
```suggestion
		//starting threads of the robot
```

**Review by konstahakli — APPROVED** [review]
Now everything looks great :)

**Review by Tuomas-Talkio — APPROVED** [review]
good job, very good

**Pull request merged**